pygen/pygen_src/test/riscv_instr_cov_test.py

```diff
@@ -15,34 +15,39 @@
 
 import sys
 import vsc
+import argparse
 import csv
 from tabulate import *
-from pygen.pygen_src.isa.riscv_cov_instr import riscv_cov_instr
-from pygen.pygen_src.riscv_instr_cover_group import *
-from pygen.pygen_src.riscv_instr_pkg import *
-
-logging.basicConfig(filename='logging_two_files_new.log', filemode='w',
-                    format="%(filename)s %(lineno)s %(levelname)s %(message)s",
-                    level=logging.ERROR)
+sys.path.append("pygen/")
+from pygen_src.isa.riscv_cov_instr import riscv_cov_instr
+from pygen_src.riscv_instr_cover_group import *
+from pygen_src.riscv_instr_pkg import *
 
 
 class riscv_instr_cov_test:
     """ Main class for applying the functional coverage test """
 
-    def __init__(self, argv):
+    def __init__(self):
         self.instr_cg = riscv_instr_cover_group()
         self.trace = {}
-        self.csv_trace = argv
+        self.csv_trace = []
         self.entry_cnt, self.total_entry_cnt, self.skipped_cnt, \
         self.unexpected_illegal_instr_cnt = 0, 0, 0, 0
+        self.argv = self.parse_args()
+        logging.basicConfig(filename='{}'.format(self.argv.log_file_name),
+                            filemode='w',
+                            format="%(filename)s %(lineno)s %(levelname)s %(message)s",
+                            level=logging.DEBUG)
 
     def run_phase(self):
+        trace_files = self.argv.trace_csv.split(",")
+        for item in trace_files:
+            self.csv_trace.append(item.split("_", 1)[1])
```

Comment:
to clarify, this is needed to remove the `_[i]` suffix, correct?

Reply:
Yes, it remove the `[i]_` prefix (although it's not needed at all, but I wanted to follow the FC on SV side).

Reply:
just left a comment above about this, but I think the indexing can be removed altogether, the SV side only uses the index to uniquely name plusargs which the cov_test.sv can pick up during runtime.

Reply:
Ah got it, that makes sense. I'll remove it then.